platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java

```diff
@@ -191,7 +184,31 @@ public LogoutResponse buildLogoutResponse(
         SamlProtocol.createIssuer(issuerOrEntityId),
         SamlProtocol.createStatus(statusCodeValue),
         inResponseTo,
-        id);
+        "a" + UUID.randomUUID().toString());
```

Comment:
✏️ We could be consistent with `SamlProtocol` and use an underscore instead of an "a". https://github.com/codice/ddf/blob/7a541112d93cc416f3391770d4d387ebd9dc6130/platform/security/platform-security-core-api/src/main/java/ddf/security/samlp/SamlProtocol.java#L243